xml/System.Net.Http/SocketsHttpHandler.xml

```diff
@@ -415,9 +437,18 @@ For example, if the value is 64, then 65,536 bytes are allowed for the maximum r
         <ReturnType>System.TimeSpan</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <summary>Gets or sets how long a connection can be the pool to be considered reusable.</summary>
+        <value>The maximum time for a connection to be in the pool. The default value for this property is <see cref="F:System.Threading.Timeout.InfiniteTimeSpan" />.</value>
+        <remarks> <format type="text/markdown"><![CDATA[
+
+## Remarks
+ This property define maximal connection lifetime in the pool regardles the connection is idle or active. The connection should be reestablished periodically to reflect the DNS TTL. There is no link between actuall DNS TTL and this property and the property shoul be set manually.
+ 
+ If the connection endpoint is not the Doamin name but the IP address, the value can be <xref:System.Threading.Timeout.InfiniteTimeSpan>.
```

Comment:
Is this based on the code? Or is this just hint? The DNS refresh matters only if server is on dynamic IP IMHO.

Reply:
It is just a hint. Please correct me there. 

I was about more static setup where the server can change its IP because infra changes and DNS reflect this change after TTL.

Any suggestion (change/ truncate the sentence) is appreciate.

Reply:
Maybe just say something like 
> This property defines maximal connection lifetime in the connectioon pool regardles the connection is idle or active. The connection will be reestablished periodically to reflect the DNS or other network changes. 

and I would probably remove the part about IP address. I think it is up to the caller to decide if refresh make sense for them or not.